conf/supervisord-dev.conf

```diff
@@ -0,0 +1,35 @@
+[supervisord]
+nodaemon = true
+silent = true
+
+[program:web]
+command=gunicorn -c conf/gunicorn/dev.conf.py --paste conf/development.ini
+stdout_events_enabled=true
+stderr_events_enabled=true
+stopsignal = KILL
+stopasgroup = true
+
+[program:nginx]
+command=docker-compose up --no-color --exit-code-from nginx-proxy nginx-proxy
+stdout_events_enabled=true
+stderr_events_enabled=true
+stopsignal = KILL
+stopasgroup = true
+
+[eventlistener:logger]
+command=bin/logger --dev
+buffer_size=100
+events=PROCESS_LOG
+stderr_logfile=/dev/fd/1
+stderr_logfile_maxbytes=0
+stdout_logfile=/dev/null
+
+[unix_http_server]
+file = .supervisor.sock
+
+[rpcinterface:supervisor]
+supervisor.rpcinterface_factory = supervisor.rpcinterface:make_main_rpcinterface
```

Comment:
I noticed that h's `conf/supervisord-dev.conf` doesn't have this section. Should it?

Reply:
Yes, it should. This section is for the `make supervisor` command which h doesn't have. It's actually not a super useful command though (I haven't found myself using it) so not a high priority. We could possibly even remove it from Via 3 and LMS